src/Elements/Spatial/Grid2d.cs

```diff
@@ -129,9 +129,9 @@ public Grid2d(IList<Polygon> boundaries, Transform t = null)
             }
 
 
-            toGrid = new Transform(t);
-            t.Invert();
             fromGrid = new Transform(t);
+            t.Invert();
```

Comment:
Transforms are passed by reference, so here you are inverting the transform passed into this method affecting everywhere else it is used. Is this the intention?

Reply:
Perhaps you mean `fromGrid.Invert()`?

Reply:
Good catch. I think this is actually doing what I wanted it to, but the possibility of side effects would have been a problem — this is corrected now. The intention is to set `fromGrid` to be the existing transform `transform`, and toGrid to be its inverse.